pulsar-broker-common/src/main/java/com/yahoo/pulsar/broker/ServiceConfiguration.java

```diff
@@ -941,4 +961,118 @@ public String getReplicatorPrefix() {
     public void setReplicatorPrefix(String replicatorPrefix) {
         this.replicatorPrefix = replicatorPrefix;
     }
+
+
+    // Configurations for new load manager API
+
+    // Number of samples to use for short term time window
+    private int loadManagerNumberOfSamplesShortTermWindow = 50;
+
+    // Number of samples to use for long term time window
+    private int loadManagerNumberOfSamplesLongTermWindow = 1000;
+
+    // How often in seconds to update the broker data
+    private long loadManagerBrokerDataUpdateIntervalInSeconds = 60;
+
+    // How often in seconds to update the bundle data
+    private long loadManagerBundleDataUpdateIntervalInSeconds = 60;
+
+    // Default throughput in to assume for new bundles
+    private double loadManagerDefaultMessageThroughputIn = 50000;
+
+    // Default throughput out to assume for new bundles
+    private double loadManagerDefaultMessageThroughputOut = 50000;
+
+    // Default message rate in to assume for new bundles
+    private double loadManagerDefaultMessageRateIn = 50;
+
+    // Default message rate out to assume for new bundles
+    private double loadManagerDefaultMessageRateOut = 50;
+
+    // Name of load manager to use
+    private String loadManagerClassName = "com.yahoo.pulsar.broker.loadbalance.impl.SimpleLoadManagerImpl";
```

Comment:
There's a bit of an issue with this, in that pulsar-broker depends on pulsar-broker-common and maven doesn't allow circular dependencies. Without package reorganization I'm not sure it can be done this way.